Document content package configuration in the README (#191)
openmrs/openmrs-module-billing: README.md

## Configuration

> The billing module can be configured through a **content package** that the [Initializer module](https://github.com/mekomsolutions/openmrs-module-initializer) applies when the server starts.

Reviewer: Should we name the minimum Initializer version here? These four domains only landed in [2.12.0](https://github.com/mekomsolutions/openmrs-module-initializer/releases/tag/2.12.0), and on anything older Initializer skips the folders silently.

```suggestion
The billing module can be configured through a **content package** that the [Initializer module](https://github.com/mekomsolutions/openmrs-module-initializer) 2.12.0 or later applies when the server starts.
```

Billing configuration lives in your content package under `configuration/backend_configuration/`:

```
configuration/backend_configuration/
├── globalproperties/billing.xml
├── billableservices/billableServices.csv
├── paymentmodes/paymentModes.csv
├── cashpoints/cashPoints.csv
└── cashieritemprices/cashierItemPrices.csv
```

> **Please make sure that the folder names are named correctly.**

Reviewer: Nit: "folder names are named" reads a bit oddly, and the tree above already shows them.

```suggestion
**The folder names must match exactly.**
```

> #### Global properties

Reviewer: ```suggestion
### Global properties
```

These are `####` but `### Assign privileges` below is `###`. Ditto for the four sections after this one.

- `billing.defaultReceiptReportId`: Jasper report ID for receipt generation
- `billing.defaultShiftReportId`: Jasper report ID for shift reports
- `billing.receipt.logoPath`: Path to receipt logo image
- `billing.systemReceiptNumberGenerator`: Class name for receipt number generator (default: `org.openmrs.module.billing.api.SequentialReceiptNumberGenerator`)

> - `billing.sequenceBlockSize`: Number of receipt sequence values reserved per database round-trip (default: 100). Larger blocks reduce database contention; smaller blocks reduce the sequence values skipped on restart (up to blockSize - 1 per group). Receipt numbers are always unique but may skip values.

Reviewer 2: Do we need this, since we're repeating it later on line 200+?

> - `billing.sequenceBlockSize`: Number of receipt sequence values reserved per database round-trip (default: 100). Larger blocks reduce database contention; smaller blocks reduce the sequence values skipped on restart (up to blockSize - 1 per group). Receipt numbers are always unique but may skip values.
>   Every `billing.*` property which you can find defined in the `config.xml` file in this repository inside a `<globalProperty>` tag, goes in `globalproperties/billing.xml`

Reviewer: ```suggestion
- `billing.sequenceBlockSize`: Number of receipt sequence values reserved per database round-trip (default: 100). Larger blocks reduce database contention; smaller blocks reduce the sequence values skipped on restart (up to blockSize - 1 per group). Receipt numbers are always unique but may skip values.

Every `billing.*` property defined inside a `<globalProperty>` tag in this repository's `config.xml` goes in `globalproperties/billing.xml`.
```

The indent makes this render inside the `sequenceBlockSize` bullet. If the list goes per @wikumChamith's comment, this sentence should stay.

| `billing.reports.paymentsByPaymentMode` | —       | ID of the Payments by Payment Mode report |

### Receipt numbering

The default generator hands out sequential receipt numbers. To avoid a database round-trip per bill it reserves a block
of `billing.sequenceBlockSize` values at a time and serves them from memory. Receipt numbers are always unique, but
values can be skipped: restarting the server discards whatever is left of the current block, losing up to
`blockSize - 1` values per sequence group. Larger blocks reduce contention under load; smaller blocks reduce the gaps.

To use your own numbering scheme, implement `org.openmrs.module.billing.api.IReceiptNumberGenerator` in another module
and set `billing.systemReceiptNumberGenerator` to its fully-qualified class name.

### Patient payment status

`billing.patientPaymentStatusResolver` selects how a patient's overall payment status is derived. Leave it blank to use
the built-in resolver, which reads existing bill records. To override it, implement
`org.openmrs.module.billing.api.PatientPaymentStatusResolver`, register your implementation as a Spring component in

> your own module so it is discoverable, and set the property to its fully-qualified class name.

Reviewer 2: Shouldn't these be outside the "Global properties reference" topic?
